### ISSUE TITLE
PandasPipeline out of range metric

### ISSUE THREAD

**Issue by elphick** [elphick/sklearn-viz#55]
PandasPipeline is a convenience class to support naming a pipeline and returning a pandas dataframe.
This task is to extend it to return a distance metric for predictions, based on how far the provided features are in N-D space to the training set.

This can be useful as a confidence indicator for the predictions.
